src/sentry/nodestore/bigtable/backend.py

```diff
@@ -209,9 +250,9 @@ def encode_row(self, id, data, ttl=None):
         # This only flag we're tracking now is whether compression
```

Comment:
nit: We always compress now so this comment can be removed.

Reply:
it's still possible to disable compression